src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -39,10 +51,67 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 <details>
 <summary>Click to see the solution</summary>
 
+✏️ Update **src/App.tsx** to be:
+@diff ../../../exercises/react-native/03-creating-a-new-app/04-solution/App.tsx ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx only
```

Comment:
The diff shows this line changing too, is that right?

```
<Text>Place My Order: Coming Soon To...</Text>
```

Reply:
Yea, it went from below. 

`<Text>Place` My Order: Coming Soon!</Text>`

Should I remove the ellipse?